content/applications/websites/elearning.rst

```diff
@@ -0,0 +1,258 @@
+to see your course front end.
+
+Course creation
+---------------
+
+Click :guilabel:`New` to create a new course. A page pops up. Add your :guilabel:`Course Title` and
+one or more :guilabel:`Tags` to describe your course. You can add an image to illustrate your course
+by clicking on the :guilabel:`Edit` option. Four tabs allow you to edit further your course:
+:ref:`Content <elearning/content>`, :ref:`Description <elearning/description>`,
+:ref:`Options <elearning/options>`, :ref:`Karma <elearning/karma>`.
+
+.. image:: elearning/elearning-course-creation.png
+   :align: center
+   :alt: Create your elearning course.
+
+.. _elearning/content:
+
+Content tab
+~~~~~~~~~~~
+
+This tab allows you to manage your course content. Click on :guilabel:`Add Section` to divide your
+course into different sections. Click on :guilabel:`Add Content` to create
+:ref:`content <elearning/create-content>`. Click on :guilabel:`Add Certification` to assess the level of
+understanding of your attendees, certify their skills and motivate them. **Certification** is part
+of the :doc:`Surveys <../marketing/surveys/overview/create>` app.
+
+.. _elearning/description:
+
+Description tab
+~~~~~~~~~~~~~~~
+
+Add a short description of your course. It appears under your course title on your website.
+
+.. image:: elearning/course-description.png
+   :align: center
+   :alt: Add a description to your course.
+
+.. _elearning/options:
+
+Options tab
+~~~~~~~~~~~
+
+This tab is divided into different sections: :ref:`Course <elearning/course>`,  :ref:`Communication
+<elearning/communication>`, :ref:`Access rights <elearning/access-rights>`, :ref:`Display
+<elearning/display>`.
+
+.. image:: elearning/options-tab.png
+   :align: center
+   :alt: Overview of the Options tab
+
+.. _elearning/course:
+
+Course
+******
+
+Set up who is :guilabel:`Responsible` for your course, and select a :guilabel:`Website` if you want
+your course to be restricted to this website only.
+
+.. _elearning/communication:
+
+Communication
+*************
+
+- :guilabel:`Allow Reviews`: tick the box to allow attendees to like and comment your content and to
+  submit reviews on your course;
+- :guilabel:`Forum`: add a dedicated forum to your course (only possible if **Forum** is installed);
+- :guilabel:`New Content Notification`: select an email template that is sent to your attendees when
+  you upload new content. Tick the green arrow to update the email template;
+- :guilabel:`Completion Notification`: select an email template that is sent to your attendees once
+  they reach the end of your course. Tick the green arrow to update the email template.
+
+.. _elearning/access-rights:
+
+Access rights
+*************
+
+- :guilabel:`Show course to`: define who can access your course and their content between
+  :guilabel:`Everyone`, :guilabel:`Signed In` or :guilabel:`Course Attendees`;
+- :guilabel:`Enroll Policy`: define how people enroll to your course. Select:
```

Comment:
If you put select here then in the bullet points before you have to say:

Open: if you want to make it available to everyone 
..
On Payment: if you select On payment, ..... 
NOTE the paid courses features must me enabled. (Maybe quickly explain how)